tests/testsuite/kubevirtresource.go

```diff
@@ -91,6 +131,13 @@ func AdjustKubeVirtResource() {
 			},
 		},
 	}
+	arch := TranslateBuildArch()
+	// Disable CPUManager Featuregate for s390x as it is not supported.
+	if arch != "s390x" {
+		kv.Spec.Configuration.DeveloperConfiguration.FeatureGates = append(kv.Spec.Configuration.DeveloperConfiguration.FeatureGates,
+			virtconfig.CPUManager,
+		)
+	}
```

Comment:
Does it really matter?

Reply:
The Kubernetes CPU Manager feature is not supported for s390x. So, our worker nodes don't have the CPU Manager label set to true. So, if we enable the feature gate for the kubevirt. When we run the e2e tests. This function https://github.com/kubevirt/kubevirt/blob/main/tests/testsuite/kubevirtresource.go#L149 is checking any available nodes with the CPU manager label and it is not able to find them and eventually failing the tests at the initial stage.